velox/connectors/Connector.h

```diff
@@ -253,6 +254,7 @@ class ConnectorQueryCtx {
       memory::MemoryPool* operatorPool,
       memory::MemoryPool* connectorPool,
       const Config* sessionProperties,
+      const core::QueryConfig* queryConfig,
```

Comment:
This is omitted by purpose.  Please forward the necessary properties in `QueryCtx::connectorSessionProperties` instead of passing the whole query config as additional argument.